bin/setup

```diff
@@ -1,29 +1,36 @@
 #!/usr/bin/env ruby
-require 'pathname'
+require 'fileutils'
+include FileUtils
 
```

Comment:
Need both of these? Or is this a 5.2 thing?

Reply:
Actually, `require` and `include` do two totally separate but rather similar things, so yes you do need both of them.

Reply:
And it is just a 5.2 thing, this is a line added by Rails.